comp/trace/config/setup.go

```diff
@@ -307,6 +306,7 @@ func applyDatadogConfig(c *config.AgentConfig, core corecompcfg.Component) error
 	if otlp.IsEnabled(coreconfig.Datadog) {
 		grpcPort = core.GetInt(coreconfig.OTLPTracePort)
 	}
+	fmt.Printf("DOING OTLP STUFF: %v\n", core.GetFloat64("otlp_config.traces.probabilistic_sampler.sampling_percentage"))
```

Comment:
Is this change expected?

Reply:
Sorry, I was troubleshooting the last changes. Thanks for the heads-up @ogaca-dd!